test/e2e/e2e_suite_test.go

```diff
@@ -67,6 +68,8 @@ var (
 var (
 	ctx = ctrl.SetupSignalHandler()
 
+	watchesCtx, cancelWatches = context.WithCancel(ctx)
```

Comment:
Let's add a short comment about what we are using this ctx for

Reply:
added a comment to both occurencies :-) PTAL

Reply:
Thx!